docs/local-deployment/overview.mdx

```diff
@@ -8,16 +8,16 @@ description: Deploy Self-hosted Aserto locally
 
 Aserto authorizers are built to be deployed right next to your application, in your cloud.
 
-## Aserto Sidecar
+## Edge Authorizer
 
-The simplest local deployment is in the form of a single container, known as the **Aserto Sidecar**. You can deploy it either as a local microservice (and scale it horizontally), or if you're running in Kubernetes, you can deploy it as a sidecar in your application pod.
+The simplest local deployment is in the form of a single Topaz container that is configured as an Aserto Edge Authorizer. You can deploy it either as a local microservice (and scale it horizontally), or if you're running in Kubernetes, you can deploy it as a sidecar in your application pod.
```

Comment:
do we want to link to www.topaz.sh? Looks like the first mention of Topaz in the deployment section